examples/message.js

```diff
@@ -0,0 +1,22 @@
+var phantom = require("../lib");
+var _ph, _page, _outObj;
+
+phantom.create().then(ph => {
+    _ph = ph;
+    return _ph.createPage();
+}).then(page => {
+    _page = page;
+    _ph.onMessage(function(resourceUrl) {
+    	console.log('Node context received resourceUrl: ', resourceUrl);
+    });
+    _page.property('onResourceRequested', function(resource, networkRequest, callbackMessagePrefix) {
+    	console.log(callbackMessagePrefix + resource.url);
```

Comment:
This feel really hacky and really anti pattern to how the rest of the code works. I don't think it would make sense for someone to have to call `_ph.getCallbackMessagePrefix()`. It should be somehow abstracted away.